ui/apps/platform/src/Containers/ComplianceEnhanced/Status/ComplianceStatusPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import { complianceEnhancedStatusPath } from 'routePaths';
+import ComplianceDashboardPage from './Dashboard/ComplianceDashboardPage';
+
+function ComplianceStatusPage() {
+    return (
+        <>
+            <Switch>
```

Comment:
What do you think about redirect from /main/compliance-enhanced list this?
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/PolicyManagement/PolicyManagementPage.tsx#L11